tests/test_model.py

```diff
@@ -321,49 +321,58 @@ def test_product_stream_builds():
     rhel_8_2_base_build = SoftwareBuildFactory()
     rhel_7_1_build = SoftwareBuildFactory()
     rhel, rhel_7, rhel_7_1, _, rhel_8_1, rhel_8_2, rhel_8_2_base = create_product_hierarchy()
+    # ProductModel.builds now uses components to lookup builds, so we need to create Components
+    # and call save_product_taxonomy on the builds to ensure we have the connection from streams
+    # to builds via components
```

Comment:
Since we only use the .builds function to get a count on the number of builds for a ProductModel I think the change here to exclude unprocessed builds from the count will have no negative impacts.